src/app.ts

```diff
@@ -41,7 +41,7 @@ export default class App {
     // Error handling
     this.app.use(serverErrorHandler);
 
-    const port = process.env.PORT || ConfigService.getIntValue('server.port');
+    const port = ConfigService.getStringValue('server.port');
```

Comment:
I think we'll also need to change the config values to string in yml files.

Reply:
done @akashlakhera  sir.